core/src/test/scala/org/apache/spark/SparkFunSuite.scala

```diff
@@ -27,19 +27,55 @@ import org.apache.spark.util.AccumulatorContext
 
 /**
  * Base abstract class for all unit tests in Spark for handling common functionality.
+ *
+ * Thread audit happens normally here automatically when a new test suite created.
+ * The only prerequisite for that is that the test class must extend [[SparkFunSuite]].
+ *
+ * There are some test suites which are doing initialization before [[SparkFunSuite#beforeAll]]
```

Comment:
Better:

"
It is possible to override the default thread audit behavior by setting `enableAutoThreadAudit` to false and manually calling the audit methods, if desired. For example:

// Code
"

Reply:
Changed.